src/ui/components/analytics-view.ts

```
@@ -0,0 +1,330 @@
+        location.counters.forEach(counter => {
+            counter.areas.forEach(area => itemsToAnalyze.push(...area.inventoryItems));
+        });
+    }
+
+    if (itemsToAnalyze.length === 0) {
+        showToast("Keine Inventurdaten im ausgewählten Bereich/Standort für die Analyse gefunden.", "info");
+        clearReportAndCharts();
+        return;
+    }
+
+    const totalConsumedItems = consumptionReport.reduce((sum, item) => sum + (item.consumedVolumeMl !== undefined && item.consumedVolumeMl > 0 ? 1 : 0), 0)
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)